src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportReview.tsx

```diff
@@ -228,8 +229,12 @@ export function SampleMetadataImportReview(): JSX.Element {
   );
 
   return (
-    <SampleMetadataImportWizard current={2} percent={progress}>
-      <Text>{i18n("SampleMetadataImportReview.description")}</Text>
+    <SampleMetadataImportWizard current={2}>
+      <Text>
```

Comment:
Does this cause a page shift?  Could you put the in the overlay under the percentage?

Reply:
Moved the text into the spinner. Not sure if I got the css right. Please see e3f82ed8c5cd14770490cd755d7fb0b4f112400f.